clap_derive/src/attrs.rs

```diff
@@ -203,8 +201,7 @@ impl Attrs {
         }
         match &*res.kind {
             Kind::Subcommand(_) => abort!(res.kind.span(), "subcommand is only allowed on fields"),
-            Kind::Skip(_) => res,
-            Kind::Arg(_) => res,
+            Kind::Skip(_) | Kind::Arg(_) => res,
```

Comment:
ditto about not grouping based on current implementation

Reply:
reverted